man/systemd-cryptenroll.xml

```diff
@@ -251,55 +256,65 @@
           <!-- See: https://github.com/tianocore-docs/edk2-TrustedBootChain/blob/main/4_Other_Trusted_Boot_Chains.md -->
           <!-- See: https://wiki.archlinux.org/title/Trusted_Platform_Module#Accessing_PCR_registers -->
 
-          <tgroup cols='2' align='left' colsep='1' rowsep='1'>
+          <tgroup cols='3' align='left' colsep='1' rowsep='1'>
             <colspec colname="pcr" />
+            <colspec colname="string_alias" />
             <colspec colname="definition" />
 
             <thead>
               <row>
                 <entry>PCR</entry>
+                <entry>alias</entry>
                 <entry>Explanation</entry>
               </row>
             </thead>
 
             <tbody>
               <row>
                 <entry>0</entry>
+                 <entry>core-firmware</entry>
                 <entry>Core system firmware executable code; changes on firmware updates</entry>
               </row>
 
               <row>
                 <entry>1</entry>
+                <entry>platform-config</entry>
                 <entry>Core system firmware data/host platform configuration; typically contains serial and model numbers, changes on basic hardware/CPU/RAM replacements</entry>
               </row>
 
               <row>
                 <entry>2</entry>
+                <entry>external-code</entry>
                 <entry>Extended or pluggable executable code; includes option ROMs on pluggable hardware</entry>
               </row>
 
               <row>
                 <entry>3</entry>
+                <entry>external-firmware</entry>
                 <entry>Extended or pluggable firmware data; includes information about pluggable hardware</entry>
               </row>
 
               <row>
                 <entry>4</entry>
+                <entry>boot-loader</entry>
                 <entry>Boot loader and additional drivers; changes on boot loader updates. The shim project will measure the PE binary it chain loads into this PCR. If the Linux kernel is invoked as UEFI PE binary, it is measured here, too. <citerefentry><refentrytitle>sd-stub</refentrytitle><manvolnum>7</manvolnum></citerefentry> measures system extension images read from the ESP here too (see <citerefentry><refentrytitle>systemd-sysext</refentrytitle><manvolnum>8</manvolnum></citerefentry>).</entry>
               </row>
 
               <row>
                 <entry>5</entry>
+                <entry>boot-loader-config</entry>
```

Comment:
hmm, this isnt right?

Reply:
> hmm, this isnt right?

"boot-gpt" you suggested, @medhefgo suggested "boot-loader-config"

How about we use "boot-gpt-config"?

Reply:
The spec says it's boot loader confg/data. The fact that the gpt is measured into it is only part of it. So putting gpt into the name is not right.